mos_tests/keystone/test.py

```diff
@@ -0,0 +1,98 @@
+#    Copyright 2016 Mirantis, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+
+import logging
+
+import pytest
+from tempest.lib.cli import output_parser as parser
+
+from mos_tests.environment.devops_client import DevopsClient
+from mos_tests.neutron.python_tests.base import TestBase
+from mos_tests import settings
+
+
+logger = logging.getLogger(__name__)
+
+
+class TestKeystone(TestBase):
+
+    @pytest.mark.testrail_id('844692')
+    @pytest.mark.parametrize('glance_remote', [2], indirect=['glance_remote'])
+    def test_keystone_bla(self):
+        """Test to cover bug#1546197
+
+        Steps to reproduce:
+
+	1. Create new project and new user there
+	2. Add new project like member in admin tenant
+	3. Login under this user
+	4. Execute 'keystone token-get' in controller
+	5. Get TOKEN_ID
+	6. Execute curl request: curl -H "X-Auth-Token: TOKEN_ID" http://192.168.0.2:5000/v2.0/tenants
+
+	ER: curl should return correct result
+
+	7. Delete new user from admin tenant
+	8. Repeat curl request
+
+	ER: curl should return 401-error
+        """
+
+        user = self.os_conn.keystone.users.create('testuser','testuser')
```

Comment:
Unfortunately, this won't work. To create users you should use os_cli.OpenStack instance